libs/protos/protos/sp_pipeline.proto

```diff
@@ -31,10 +37,14 @@ message Pipeline {
   // One or more steps to execute
   repeated PipelineStep steps = 3;
 
+
+
   // Notification configs for this pipeline. Only filled out in external API responses
   // This is deprecated and the data has moved to PipelineStep
   repeated protos.NotificationConfig _notification_configs = 4 [deprecated=true]; // protolint:disable:this FIELD_NAMES_LOWER_SNAKE_CASE
 
+  PipelineDataFormat data_format = 5;
```

Comment:
Should we put this on `Step` instead? Is there a reason why this needs to be on the Pipeline?

Reply:
The data type isn't going to change between steps, it needs to apply to them all, otherwise we'll have to specify individually for each step